services/checks/checks.go

```diff
@@ -138,22 +139,12 @@ func (s *Service) Run(ctx context.Context) {
 	defer ticker.Stop()
 
 	for {
-		var sttEnabled bool
-		settings, err := models.GetSettings(s.db)
-		if err != nil {
-			s.l.Error(err)
-		}
-		if settings != nil && settings.SaaS.STTEnabled {
-			sttEnabled = true
-		}
-
-		if sttEnabled {
-			nCtx, cancel := context.WithTimeout(ctx, checksTimeout)
-			s.collectChecks(nCtx)
-			s.executeChecks(nCtx)
-			cancel()
-		} else {
-			s.l.Info("STT is not enabled, doing nothing.")
+		if err := s.StartChecks(ctx); err != nil {
+			if err == services.STTDisabledError {
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
err113: do not compare errors directly, use errors.Is() instead: "err == services.STTDisabledError" (goerr113)